resources/views/components/sidebar-link-content-mod.blade.php

```diff
@@ -20,7 +20,7 @@ class="absolute bottom-auto left-0 right-auto top-0 z-10 inline-block -translate
                     {{ $badgeText }}
                 </div>
             @endif
-       <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" aria-hidden="true" class="w-6 h-6 stroke-{{ $roleColour }}-400">
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" aria-hidden="true" class="w-6 h-6 {{ $isCurrentRoute ? 'stroke-white' : 'stroke-' . $roleColour . '-400' }}">
```

Comment:
Again refer to https://tailwindcss.com/docs/content-configuration#dynamic-class-names